### DISCUSSION TITLE
教程：升级到7.8.4，解决蓝灯闪退的问题

### DISCUSSION THREAD

**Discussion by mqingqing123** [getlantern/lantern-forum-cn#421] [General]
本文详细介绍如何升级蓝灯到 7.8.4。

##  （1）下载 7.8.4 版
点击 https://github.com/getlantern/lantern 在右侧有一个“release（发布版）”，找到 7.8.4

![image](https://github.com/getlantern/lantern-forum-cn/assets/22250695/3fe11fe7-8803-459d-b082-a86317aa6ba7)

## （2）windows版本，下载  update_windows_386.bz2

![image](https://github.com/getlantern/lantern-forum-cn/assets/22250695/b7c4dfd0-b8dd-4994-b510-0ced7227ad7e)

## （3）解压后得到  update_windows_386，在其上，右键选中重命名，增加 .exe 扩展名，即 “ update_windows_386.exe ”
双击，即可打开蓝灯7.8.4，双击即可运行

![image](https://github.com/getlantern/lantern-forum-cn/assets/22250695/491659b9-c0cc-46e5-9517-2b821108756b)

 ##   （4）原先的账户，继续有效 
![image](https://github.com/getlantern/lantern-forum-cn/assets/22250695/39cf7084-b3de-457e-8518-b96134d4ea5d)

 ###   一点提示：
（1）7.8.4 可能是官网的临时解决方案。 
（2）在蓝灯上，右键选中打开文件夹所在方式

（3）**global.yaml** 是用户的账号密钥，请务必备份好此文件，
如果修改此文件，蓝灯会认为你是免费会员。

![image](https://github.com/getlantern/lantern-forum-cn/assets/22250695/5b5015ef-051e-46a1-b573-23bc143aa4f4)

**Comment by mqingqing123** [comment]
请保存了 global.yaml 文件

**Comment by MaxMKMax** [comment]
@mqingqing123 谢谢您的教学. 在下崇拜与感激您!
